tests/unit/Exports/OrderExportTest.php

```diff
@@ -721,14 +824,14 @@ protected function processNewOrder(int $new_order_id, array $new_order): void
         $task = $OrderHandler->create($new_order_id);
 
         // set the checker for expected result
-        $sk_order_id = rand();
-        $sk_customer_id = rand();
+        $sk_order_id = mt_rand();
+        $sk_customer_id = mt_rand();
 
         $sent_order = [];
         StoreKeeperApi::$mockAdapter->withModule(
             'ShopModule',
             function (MockInterface $module) use ($sk_customer_id, $sk_order_id, $new_order, $new_order_id, &$sent_order) {
-                $module->shouldReceive('newOrder')
+                $module->allows('newOrder')
```

Comment:
why was the change to allows?

Reply:
It's legacy Mockery syntax, so I just applied where I can for a bit improvement.

Reply:
than it should exepects